target/linux/ramips/image/mt7620.mk

```diff
@@ -894,10 +894,40 @@ define Device/ravpower_wd03
   IMAGE_SIZE := 7872k
   DEVICE_VENDOR := Ravpower
   DEVICE_MODEL := WD03
-  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci
+  DEVICE_PACKAGES := kmod-usb2 kmod-usb-ohci kmod-i2c-core kmod-i2c-ralink
+  LOADER_TYPE := bin
+  LOADER_FLASH_OFFS := 0xFD051000
+  COMPILE := loader-$(1).bin
+  COMPILE/loader-$(1).bin := loader-okli-compile | pad-to 64k | lzma | \
+        uImage lzma
+#  KERNEL := $(KERNEL_DTB) | uImage lzma -M 0x4f4b4c49 | loader-okli $(1) 4096
+  KERNEL := $(KERNEL_DTB) | uImage lzma -M 0x4f4b4c49
+  KERNEL_INITRAMFS := $(KERNEL_DTB) | uImage lzma
+  IMAGES += factory.bin
+  IMAGE/factory.bin := append-kernel | loader-okli $(1) 4096 | \
+		append-rootfs | pad-rootfs | check-size
 endef
 TARGET_DEVICES += ravpower_wd03
 
+define Device/hootoo_tm05
+  SOC := mt7620n
+  IMAGE_SIZE := 7872k
```

Comment:
Firmware partition length decreased just a little bit, so this size would overrun just a little. Actual size of "firmware" space is now (0x7af) 7868k.

Alternatively round down to (0x7a) or 122 blocks to ensure there is room for the overlayfs. 122 * 64 = 7808k

Reply:
Changed to 7808k, thanks!

Reply:
FYI, if you end up removing loader partition and it works, you can revert this